cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -210,6 +210,7 @@ func (dc *DownstreamController) syncEdgeNodes() {
 			case watch.Added:
 				fallthrough
 			case watch.Modified:
+				coreInformer := informers.GetInformersManager().GetK8sInformerFactory().Core().V1()
```

Comment:
I think it would be better to add serviceLister as a fields in DownStream struct.

Reply:
done